pyomo/contrib/mindtpy/config_options.py

```diff
@@ -523,7 +523,7 @@ def check_config(config):
         if config.mip_solver not in {'cplex_persistent', 'gurobi_persistent'}:
             config.mip_solver = 'cplex_persistent'
             config.logger.info(
-                'Only cplex_persistent and gurobi_persistent are supported for LP/NLP based Branch and Bound method. The MIP solver has been changed to cplex_persistent.')
+                'Only cplex_persistent, gurobi_persistent are supported for LP/NLP based Branch and Bound method. The MIP solver has been changed to cplex_persistent.')
```

Comment:
What if the user does not have CPLEX installed? Is this a safe choice to fallback to use CPLEX? I'm inclining more for MindtPy to stop right there and tell the user that either cplex or gurobi persisted need to be used.